conda-forge.yml

```
@@ -1,6 +1,13 @@
+build_platform:
+  osx_arm64: osx_64
+  linux_aarch64: linux_64
```

```suggestion
  linux_aarch64: linux_aarch64
```

Seems like you might need to build shiboken2 for the native platform or something like that. I say don't cross compile for now if you don't have to.